reconcile/test/test_utils_slack_api.py

```diff
@@ -167,6 +168,35 @@ def test_chat_post_message_missing_channel(slack_api):
         slack_api.client.chat_post_message('test')
 
 
+def test_join_channel_missing_channel(slack_api):
+    """Don't raise an exception when the channel is set."""
+    slack_api.client.channel = None
+    with pytest.raises(ValueError):
+        slack_api.client.join_channel()
+
+
+@patch.object(SlackApi, 'get_channels_by_names')
```

Comment:
I'd suggest replacing this with the `mocker` fixture as per: https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-testing-guide/#pytest-best-practices

It should be a simple change because `mocker` is just exposing much of the `unittest.mock` API via a pytest fixture.